config/etd.yml

```diff
@@ -65,8 +65,15 @@ custom_nodes:
       name: admin_set_id
       property: "%{work_type}.%{field_name}"
       type: String
-    method: Mapping::AdminSet.unprocessed
-    value_from_node_property: admin_set_id
+    method: Mapping::Admin_Set.lookup_admin_set
+    value_from_node_property: owner_id
+  collection:
+    field:
+      name: dspace_collection
+      property: "%{work_type}.%{field_name}"
+      type: Array
+    method: Mapping::Collection.lookup_collection_names
```

Comment:
the configuration here needs to change to `Mapping::Collection.lookup_collection_name` to match the code.